packages/oneko.rb

```diff
@@ -3,48 +3,26 @@
 class Oneko < Package
   description 'A cat chases around your mouse cursor.'
   homepage 'http://www.daidouji.com/oneko/'
-  version '1.2.sakura.6'
+  @_ver = '1.2.sakura.6-15'
+  version @_ver
   license 'public-domain'
   compatibility 'all'
   source_url 'https://httpredir.debian.org/debian/pool/main/o/oneko/oneko_1.2.sakura.6.orig.tar.gz'
   source_sha256 'd89cee8b81cdb40ef23b3457c9a7fe1b0ff130081b21a41ec6c41cda01391d25'
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/oneko/1.2.sakura.6_armv7l/oneko-1.2.sakura.6-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/oneko/1.2.sakura.6_armv7l/oneko-1.2.sakura.6-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/oneko/1.2.sakura.6_i686/oneko-1.2.sakura.6-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/oneko/1.2.sakura.6_x86_64/oneko-1.2.sakura.6-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '18128ddab7d13999c43c4fa2ce24fdd15a9bcd4541a815ad8c19f9154d006db8',
-     armv7l: '18128ddab7d13999c43c4fa2ce24fdd15a9bcd4541a815ad8c19f9154d006db8',
-       i686: '68469cb17415e9088f966a3b6551b52dcfcd15f3cec905e91fa4c7eca16dc3a0',
-     x86_64: 'abcfe89e40fb13c5df22abcbae616892d9f9917ef931520a80dbbd067bb0775d',
-  })
-
   depends_on 'libx11'
   depends_on 'libxext'
   depends_on 'sommelier'
   depends_on 'imake' => :build
   depends_on 'xorg_cf_files' => :build
   depends_on 'gccmakedep' => :build
 
+  extra_url "https://httpredir.debian.org/debian/pool/main/o/oneko/oneko_#{@_ver}.debian.tar.xz" => '8f12f3e167f100e0fcef5185f6f1faf47cb627d96c7011e580348f5f317e76b4'
+  extra_url 'https://bouncer.gentoo.org/fetch/root/all/distfiles/oneko-1.2-cat.png' => '994dec71c4021f4e228b8c69fcefde5b11244445ff8ff6d43b3790beecef5800'
+  extra_url 'https://bouncer.gentoo.org/fetch/root/all/distfiles/oneko-1.2-dog.png' => '5ca2d41b5538618196cfe9e40dd75856fde77fe35cba99ed9b2eacf1fb0e5502'
```

Comment:
Would this structure work?
```
extra_url {(
  "https://httpredir.debian.org/debian/pool/main/o/oneko/oneko_#{@_ver}.debian.tar.xz" => '8f12f3e167f100e0fcef5185f6f1faf47cb627d96c7011e580348f5f317e76b4',
  'https://bouncer.gentoo.org/fetch/root/all/distfiles/oneko-1.2-cat.png' => '994dec71c4021f4e228b8c69fcefde5b11244445ff8ff6d43b3790beecef5800',
  'https://bouncer.gentoo.org/fetch/root/all/distfiles/oneko-1.2-dog.png' => '5ca2d41b5538618196cfe9e40dd75856fde77fe35cba99ed9b2eacf1fb0e5502'
)}
```

Reply:
This would be along the lines of the recent edits enabling this format for `source_url` and `source_sha256` in package.rb. I think this is a cleaner look because it takes up less horizontal space.